tests/csapi/txnid_test.go

```diff
@@ -0,0 +1,40 @@
+package csapi_tests
+
+import (
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/client"
+	"github.com/matrix-org/complement/runtime"
+	"github.com/tidwall/gjson"
+	"testing"
+)
+
+// TestTxnInEvent checks that the transaction ID is present when getting the event from the /rooms/{roomID}/event/{eventID} endpoint.
+func TestTxnInEvent(t *testing.T) {
+	// Dendrite implementation is broken
```

Comment:
If you could file an issue (and reference it here) that would be lovely!

Reply:
Done!